mobify/spline
# Add icon-address function

Adding this missing icon function that can help make building icon fonts more easy.

For example this will be used to build icon components by looping through an $icon-list.

Status: **Opened for visibility**
Reviewers: @kpeatt @ry5n
## Changes
- Add `_icon-address.scss` file to `/functions` directory
- Remove older, out of date `_icon-font-content.scss` file
- Import `icon-address` into `_spline.scss`
- Minor improvements to Error messaging
## How to Test
1. Clone this repo to your machine
2. Checked out this PR branch (`add-icon-address-function`)
3. Navigate to `/spline/test`
4. Compile the CSS by running `sass --update .`
5. Serve the `/spline` using `serve` ([see here](http://get-serve.com/) if you don't have this tool)
6. Visit the serve directory in your browser (i.e. `http://localhost:4000/`)
7. Scroll to "Icon Font Test" the icons should render without issue
